templates/data-providers/result-card.jsx

```diff
@@ -17,19 +18,19 @@ const ResultCard = ({
     <span className={styles.resultHeading}>
       <Icon src={icon} aria-hidden />
       <Link
-        className={styles.resultTitle}
-        target="_blank"
-        href={`https://core.ac.uk/search?q=repositories.id:(${repoId})`}
+        as={`/data-providers/${repoId}`}
+        href="/data-providers/[data-provider-id]"
+        passHref
       >
-        {title}
+        <UILink className={styles.resultTitle}>{title}</UILink>
       </Link>
     </span>
 
     <ul className={styles.resultMetadata}>
       <li>
-        <Link href={homePage} external icon={false}>
+        <UILink href={homePage} external icon={false}>
```

Comment:
What is this?

Reply:
This is link to the homepage of repository displayed data providers search page.